src/vs/workbench/api/common/extHostStatusBar.ts

```diff
@@ -164,6 +174,11 @@ export class ExtHostStatusBarEntry implements vscode.StatusBarItem {
 		this.update();
 	}
 
```

Comment:
As suggested, I would fold this into `tooltip` as overload and then it needs proposed API checks in place.